### DISCUSSION TITLE
Querying Arrow IPC

### DISCUSSION THREAD

**Discussion by asbisen** [duckdb/duckdb#3320] [Ideas]
Hi There, 

Are there any plans to be able to query arrow IPC files, making something like `arrow_scan()` possible? With compression enabled the size of arrow files are much smaller, and the read performance of arrow files seems faster.

Thanks,

Anand

**Comment by Alex-Monahan** [comment]
Hello! 
What is the main difference between what you are interested in and the Arrow support we already have? Is it the IPC format specifically?
[Arrow integration blog post](https://duckdb.org/2021/12/03/duck-arrow.html)

Also, which client (Python, R, etc.) Are you interested in?

**Reply by asbisen** [reply]
Yes that's exactly what I am interested in, unfortunately I don't think this works in Julia client yet. Thanks.
